webprojects/SUMO.md

```diff
@@ -0,0 +1,77 @@
+
+SUMO consists of two parts:
+* User interface (or UI), such as navigation and buttons.
+* Support articles, either use to troubleshoot issue or as knowledge base for Mozilla products.
+
+This document only covers the UI portion localizable in Pontoon.
+
+## Localizing the strings
+
+There are two active components in Pontoon that make up the [SUMO product](https://pontoon.mozilla.org/sumo):
+
+* **LC_MESSAGES/djangojs.po** contains strings for the JS bits of the platform.
+* **LC_MESSAGES/django.po** contains the majority of strings in the SUMO product.
+
+SUMO is made up by several file types. In order to minimize revision and get translation right the first time, the key is to get the context right. Look for clues or ask others for help:
+* [String ID](https://github.com/mozilla-l10n/sumo-l10n/blob/master/en_US/LC_MESSAGES/django.po#L265) shows where the strings is used.
+* [Comment to a string](https://github.com/mozilla-l10n/sumo-l10n/blob/master/en_US/LC_MESSAGES/django.po#L9274).
+* [The SUMO l10n forum](https://support.mozilla.org/forums/l10n-forum). But first thing first, you need to [create a SUMO account](https://support.mozilla.org/users/authcontributor) in order to participate in the discussions in the forum.
+
+Localized strings in Pontoon will be synced up with sumo-l10n on GitHub every 20 minutes, making the repository the source of truth.
+
+Please ignore these legacy components:
+
+* LC_MESSAGES/buddyup.po contains strings for mobile-centric 1-to-1 support project.
+* LC_MESSAGES/yaocho.po contains strings for the mobile version of the support knowledge base.
+
+## Testing
+
+On Thursdays, SUMO developers perform synchronization so localized content is ready on staging for testing.
+
+On Mondays, SUMO developers perform synchronization so:
+
+* New strings from code and database edits go into sumo-l10n repository.
+* Translated strings from sumo-l10n are pushed to production.
+
+There are three ways to test the product:
+
+* Staging server: https://support.allizom.org​.
+* Production server: http://support.mozilla.org/.
+* Locally: if you want to test your update in a fast and independent way, you should [install your own local instance of Kitsune](https://kitsune.readthedocs.io/en/latest/localization.html#testing-localized-strings) on your computer.
+
+### Testing on production
+
+Localization updates are pushed on Thursdays on staging for review in context, and on Mondays on production. Either way, the SUMO team relies on you to find issues and report bugs that are usually fixed at a timely manner.
+
+### Testing locally
+
+According to SUMO developers, Kitsune is in a very fragile state. The SUMO team has very limited resources to support local installs. Unless you are technical and self sufficient, you are strongly advised to use Pontoon and staging server to preview your localization work.
```

Comment:
(d'oh at the first paragraph)
Maybe rewrite: "The Kitsune codebase is currently not ready for a large audience, and the SUMO team has very limited resources to support local installs."